lib/jsonize.py

```diff
@@ -58,7 +58,7 @@ def load_exploits(path, node="exploits"):
             try:
                 selected_file = file_list[int(action) - 1]
                 selected = True
-            except Except:
+            except Exception:
```

Comment:
`except:`

Works fine in this case, unless you want to capture the error message in a var, then you can do;

```
except Exception as e:
  # Useful for handling and debugging.
  print e 
  
# In example
```